### ISSUE TITLE
Double Provider Module crash.

### ISSUE THREAD

**Issue by JannikB99** [RS485/LogisticsPipes#874]
FTB Infinity evolved 2.1.2 server crashes after putting a second Provider Module mk.2 in a chassis mk.2 pipe (needed faster extraction of items).
Everytime the chunk (with named pipe setup) loads the server crashes again.
Im sorry but Im unable to provide a log thanks to my awesome hoster (not), that said it should be easy to reproduce, gonna check newer versions too.

**Comment by SonOfTheStars** [comment]
Youre not supposed to put Multiple Providers in the Same Parent Pipe ATM.
Different Pipes will solve kinda your problem...I THINK

Though LP is really not supposed to have 2 Providers on the Same Storage Unit

**Comment by JannikB99** [comment]
Yeah but still for people with about a hundred recyclers(each has 2 overclockers) the speed simply wasnt enough so i've put two of the in (hoping it would speed things up) and well to save this mess (or rather my hours of work) i needed to go through some real messy nbt editing.
That said i would just implement a check for stuff like that to prevent putting a second one in (and ruining your chunk)

**Comment by SonOfTheStars** [comment]
The problem lies in how the Pipes aka. The Modules access the Inventory. If its 2 different Pipes you'd probably experience "Less" issues but wont be completely free. LP is not build to be extracting from 2 different sources simultaniously.

Additionally LP dev currently is in somewhat of a halt. so there wont be changes rather soon. Im not sure whats up with the Orig. Devs but something takes their time away. Most certainly Realife.

**Comment by hron84** [comment]
@Bitterholz Around Christmas it's normal I think. There's no too much cubic person like me who comments on GitHub at Christmas Day.
